dev-guide/breaking-changes.md

```diff
@@ -0,0 +1,129 @@
+---
+Title: Policy for Breaking Changes
+authors:
+  - "@dhellmann"
+  - "@deads2k"
+  - "@derekwaynecarr"
```

Comment:
A few others had a hand, but I lost track. Let me know if I should add anyone here.